scripts/bulk-index.js

```diff
@@ -0,0 +1,289 @@
+/**
+ *  Bulk Index
+ *
+ *  This script allows you to index lots of records in bulk based on an
+ *  arbitrary Bib- or ItemService SQL query.
+ *
+ *  Usage:
+ *    node scripts/bulk-index --type (item|bib) [--hasMarc MARC] [--nyplSource NYPLSOURCE]
+ *
+ *  Arguments:
+ *    type {string}: Required. One of item or bib
+ *    hasMarc {string}: Marc tag that must be present in the record
+ *    nyplSource {string}: NYPL Source value. Default 'sierra-nypl'
+ *    orderBy {string}: Columns to order query by. e.g. `--orderBy id`. Default ''
```

Comment:
can you link or outline the schema of the database somewhere (maybe a readme) so a user would know what options there are to order by?

Reply:
Added note about other possible values.